tensorflow/compiler/mlir/lite/transforms/prepare_tf.cc

```diff
@@ -746,8 +746,13 @@ struct ConvertFusedBatchNorm : public OpRewritePattern<TF::FusedBatchNormOp> {
                              tf_fused_batch_norm_op.getAttrs());
     Operation *tf_fused_batch_norm_op_v3 = rewriter.createOperation(new_state);
 
-    rewriter.replaceOp(tf_fused_batch_norm_op,
-                       tf_fused_batch_norm_op_v3->getResults());
```

Comment:
We can still use replaceOp by using take_front or drop_back method on getResults().